stdlib/sql/to_test.go

```diff
@@ -1,9 +1,10 @@
 package sql_test
 
 import (
-	"github.com/influxdata/flux/plan"
 	"testing"
 
+	"github.com/influxdata/flux/plan"
+
```

Comment:
Extra newline here. Thanks for moving it under the `testing` import though. This just needs to be grouped with the other imports below it.